examples/sampling/graphbolt/rgcn/hetero_rgcn.py

```diff
@@ -554,7 +554,12 @@ def train(
 
 def main(args):
     device = torch.device(
-        "cuda" if args.num_gpus > 0 and torch.cuda.is_available() else "cpu"
+        "cuda"
```

Comment:
OK. This seems to be a bug, because the memory of GPU cannot cantain the large graph. Maybe you can reproduce it by running: 
```
python3 examples/sampling/graphbolt/rgcn/hetero_rgcn.py --dataset ogb-lsc-mag240m
```
I will discard my change to it, and may modify it in another PR if needed.